Resources/Prototypes/Entities/Structures/Shuttles/thrusters.yml

```
@@ -102,6 +102,13 @@
       visible: false
       offset: 0, 1
 
+- type: entity
+  parent: Thruster
+  id: XenoborgThruster
```

```suggestion
  parent: Thruster
  id: ThrusterXenoborg
```

This is primarly a Thruster. What kind? Xenoborg. So ThrusterXenoborg